src/scss/components/form/_forms.scss

```diff
@@ -0,0 +1,71 @@
+@charset "UTF-8";
+// Forms
+// - - - - - - - - - - - - - - - - - - - - - - - - -
+
+fieldset {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  // Chrome and Firefox set a `min-width: min-content;` on fieldsets,
+  // so we reset that to ensure it behaves more like a standard block element.
+  min-width: 0;
+}
+
+legend {
+  @include marg(0 0 default 0);
+  display: block;
+  width: 100%;
+  color: config-get("legend-color");
+  border: 0;
+  border-bottom: 1px solid config-get("legend-border-color");
+}
+
+label {
+  @include marg(0 0 default 0);
+  display: inline-block;
+  max-width: 100%;
+  font-weight: bold;
+}
+
+#{$all-text-inputs} {
+  @include pad(rem(10) rem(10) rem(10) rem(10));
+  @include marg(0 0 default 0);
+  width: 100%;
```

Comment:
Properties should be ordered border-radius, box-shadow, display, outline, transition, width